pr-reminders/slack.py

```diff
@@ -10,24 +10,40 @@ class Bot:
     def __init__(self, slack_bot_token: str):
         self.client = WebClient(slack_bot_token)
 
-    def send_message(self, channel, message):
+    def send_message(self, channel, message, header):
         response = self.client.chat_postMessage(
             channel=channel,
-            text=message
+            blocks=[
+                {
+                    "type": "section",
+                    "text": {
+                        "type": "mrkdwn",
+                        "text": f"*_# {header}_*"
```

Comment:
Do you have an example of what the separate section looks like?

Reply:
![image](https://github.com/user-attachments/assets/bc643a75-514e-4ab8-9e15-57bd640214c2)